jockeyJSAndroid/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Having this as an additional project that generates an individual aar (I also suspect this should end up as a jar and not an aar) seems like a little bit of an overkill.
This isn't really an individual package that we encourage people to use outside of our SDK, and I suspect this will also cause some grief when utilizing the SDK from maven.
The only reason to do this is if it's considered "proper", and I suspect @MaTriXy is just the man to answer that question.